src/test/testRunner.ts

```diff
@@ -12,7 +12,7 @@ import * as fs from 'fs-extra'
 /**
  * @param initTests List of relative paths to test files to run before all discovered tests.
  */
-export async function runTestsInFolder(testFolder: string, initTests: string[] = []): Promise<void> {
+export async function runTests(testFolder: string, initTests: string[] = [], testFiles?: string[]): Promise<void> {
```

Comment:
I added this as a debugging tool when I was running into the issue of different tests interfering with each other. Justin recommended I keep it in there as a debugging tool in case a similar issue ever pops up. I'm happy to provide documentation on how to use it and how it works. How should I go about doing this? I can also drop it if it's a hassle or potentially causes issues.

Reply:
You can just leave it and I'll add a launch config parameter. It'll be easier than trying to document it through code changes.